schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,623 @@
+        "description": "An unstructured reference string.",
+        "type": "string",
+        "minLength": 1
+      },
+      "source_info": {
+        "title": "Source information",
+        "description": "Information about the source input.",
+        "type": "object",
+        "properties": {
+          "source_format": {
+            "title": "Source format",
+            "type": "string",
+            "enum": ["tex", "pdftex", "pdf", "ps", "html", "docx", "none"]
+          },
+          "legacy_source_flags": {
+            "title": "Legacy source flags",
+            "type": "string",
+            "enum": ["1", "1A", "1S", "A", "AD", "AS", "D", "S", "S1"]
+          },
+          "size_kilobytes": {
+            "title": "Source size in kilobytes",
+            "description": "The size of the source file, in kilobytes.",
+            "type": "number"
+          }
+        },
+        "required": [
+          "source_format",
+          "size_kilobytes"
+        ],
+        "additionalProperties": false
+      },
+      "titles": {
+        "title": "List of titles",
+        "description": "The English and non-English versions of the title. The English title must be provided first.",
+        "type": "array",
+        "items": [
+          {
+            "type": "object",
+            "properties": {
+              "title": {
+                "type": "string",
+                "minLength": 1
+              },
+              "language": {
+                "const": "en"
+              }
+            },
+            "additionalProperties": false
+          }
+        ],
+        "additionalItems": {
+          "type": "object",
+          "properties": {
+            "title": {
+              "type": "string",
+              "minLength": 1
+            },
+            "language": {
+               "allOf": [
+                 {"$ref": "#/definitions/language"},
+                 {"not": {"enum": ["en"]}}
+             ]
+            }
+          },
+          "additionalProperties": false
+        }
+      },
+      "author_link": {
+        "tile": "Author link",
+        "description": "Author-provided link to related resource.",
+        "type": "object",
+        "properties": {
+          "uri": {
+            "type": "string",
+            "format": "uri"
+          },
+          "display_text": {
+            "description": "Descriptive text to associate with the URI.",
+            "type": "string",
+            "minLength": 1
+          },
+          "category": {
+            "title": "Link category",
+            "description": "The type of resource the link resolves to.",
+            "type": "string",
+            "$comment": "TODO: refine this enumeration",
+            "enum": ["related article", "slides", "notes", "software", "dataset", "simulation", "video", "review", "news item", "personal website", "blog entry", "other"]
+          }
+        },
+        "required": [
+          "uri",
+          "category"
```

Comment:
Considering the problems with URLs in the abstract, maybe we should require display_text?